qhana_plugin_registry/tasks/plugin_filter.py

```diff
@@ -89,14 +89,17 @@ def get_plugins_from_filter(
             for p_id, p in plugin_mapping.items():
                 if plugin_id == p.plugin_id:
                     plugin_ids.add(p_id)
-                elif plugin_id.split("@") == p.plugin_id.split("@")[:-1]:
+                elif (
+                    plugin_id.split("@") == p.plugin_id.split("@")[:-1]
+                ):  # id matches, except for the version string
```

Comment:
If the autoformatter breaks a line like this it can be better to move the comment to the next line (into the else block)